src/features/MentorPage/components/MentorList/MentorCard/List/index.tsx

```diff
@@ -65,20 +65,20 @@ const Container = styled.div<{ isMobile: boolean }>`
           flex-wrap: wrap;
           margin: ${spacing.layout_spacing};
           max-width: calc(
-            ((76vw + ${spacing.layout_spacing} * 2) / 3) -
+            ((${CONTENT_WIDTH} + ${spacing.layout_spacing} * 2) / 3) -
               (${spacing.layout_spacing} * 2)
           );
           @media screen and (min-width: 2550px) {
             flex: 0 0 20%;
             max-width: calc(
-              ((76vw + ${spacing.layout_spacing} * 2) / 5) -
+              ((${CONTENT_WIDTH} + ${spacing.layout_spacing} * 2) / 5) -
                 (${spacing.layout_spacing} * 2)
             );
           }
           @media screen and (min-width: 1950px) {
             flex: 0 0 25%;
             max-width: calc(
-              ((76vw + ${spacing.layout_spacing} * 2) / 4) -
+              ((${CONTENT_WIDTH} + ${spacing.layout_spacing} * 2) / 4) -
```

Comment:
Good job of getting rid of these magic numbers

Reply:
Yes, at least a few of them!